src/alert/alert.style.tsx

```diff
@@ -157,12 +158,17 @@ export const TextWrapperContainer = styled.div<TextWrapperContainerStyleProps>`
     display: flex;
     flex-direction: column;
     flex: 1;
-
+    ${(props) => {
+        if(props.$showMore && props.$hasActionLink)
+            return css`
+            margin-bottom: 0.5rem;
+        `;
+    }}
     ${(props) => {
         const gradient =
             "linear-gradient(to bottom, black 50%, transparent 100%)";
         if (!props.$showMore && props.$maxCollapsedHeight)
-            return `
+            return css`
```

Comment:
there's no real need to use `css` here as we're returning plain strings, we usually only need to do so for those containing nested functions or themes